sdks/python/apache_beam/pipeline_test.py

```diff
@@ -960,6 +963,159 @@ def annotations(self):
             transform.annotations['proto'], some_proto.SerializeToString())
     self.assertEqual(seen, 2)
 
+  def test_runner_api_roundtrip_preserves_resource_hints(self):
+    resources._KNOWN_HINTS.update(
+        {'foo_hint': lambda value: {
+            'foo_urn': value
+        }})
+
+    p = TestPipeline()
```

Comment:
FWIW, there's no need to use TestPipeline here. It's mostly for ValidatesRunner integration tests.

Reply:
Thanks, fixing, must have copied from some other test.